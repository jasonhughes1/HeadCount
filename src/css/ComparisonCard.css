.comparison-center {
  display: flex;
  flex-direction: column;
  height: 285px;
  justify-content: space-around;
  left: calc(50% - 130px);
  position: absolute;
  top: 0;
  width: 263px; }
  .comparison-center h3 {
    color: #1c6d9c;
    font-size: 50px;
    text-align: center; }
  .comparison-center h4 {
    color: #1c6d9c;
    font-size: 20px;
    line-height: .9;
    opacity: .6;
    text-align: center; }
  .comparison-center h5 {
    color: #004973;
    font-family: 'Magallanes', sans-serif;
    font-size: 20px;
    text-align: center; }
  .comparison-center h6 {
    color: #1c6d9c;
    font-family: 'Magallanes', sans-serif;
    opacity: .7;
    text-align: center; }
  .comparison-center .arrow-left {
    left: -29px;
    position: absolute;
    top: 5px;
    width: 296px;
    z-index: -2; }
  .comparison-center .arrow-right {
    bottom: 5px;
    position: absolute;
    right: -29px;
    width: 296px;
    z-index: -2; }

@media screen and (max-width: 818px) {
  .comparison-center {
    height: initial;
    margin: 20px 0;
    position: static;
    width: initial; }
    .comparison-center h3, .comparison-center h6 {
      color: #fff; }
  .avg-diff {
    background: linear-gradient(to right, rgba(166, 22, 8, 0) 0%, #a61608 45%, #a61608 55%, rgba(166, 22, 8, 0) 100%);
    margin: 20px 0;
    padding: 8px 0 15px; }
  .arrow-right, .arrow-left {
    display: none; } }

@media screen and (max-width: 545px) {
  .avg-diff {
    background-color: #a61608; } }
