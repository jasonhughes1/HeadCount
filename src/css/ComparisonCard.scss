.comparison-center {
  display: flex;
  flex-direction: column;
  height: 285px;
 	justify-content: space-around;
  left: calc(50% - 130px);
  position: absolute;
  top: 0;
  width: 263px;

  h3 {
  	color: #1c6d9c;
    font-size: 50px;
    text-align: center;
  }

  h4 {
  	color: #1c6d9c;
    font-size: 20px;
    line-height: .9;
    opacity: .6;
    text-align: center;
  }

  h5 {
    color: #004973;
    font-family: 'Magallanes', sans-serif;
    font-size: 20px;
    text-align: center;
  }

  h6 {
  	color: #1c6d9c;
    font-family: 'Magallanes', sans-serif;
  	opacity: .7;
    text-align: center;
  }

  .arrow-left {
  	left: -29px;
    position: absolute;
    top: 5px;
  	width: 296px;
  	z-index: -2;
  }

  .arrow-right {
  	bottom: 5px;
    position: absolute;
  	right: -29px;
  	width: 296px;
  	z-index: -2;
  }
}

@media screen and (max-width: 818px) {
  .comparison-center {
    height: initial;
    margin: 20px 0;
    position: static;
    width: initial;

    h3, h6 {
      color: #fff;
    }
  }

  .avg-diff {
    background: linear-gradient(to right, rgba(166,22,8,0) 0%,rgba(166,22,8,1) 45%,rgba(166,22,8,1) 55%,rgba(166,22,8,0) 100%);
    margin: 20px 0;
    padding: 8px 0 15px;
  }

  .arrow-right, .arrow-left {
    display: none;
  }
}

@media screen and (max-width: 545px) {
  .avg-diff {
    background-color: rgba(166,22,8,1);
  }
}