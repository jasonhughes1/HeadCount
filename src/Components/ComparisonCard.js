import React from 'react';
import '../css/ComparisonCard.css';

const ComparisonCard = ({ loc1, avg1, loc2, avg2, compare }) => {

	return (
	  <div className="comparison-center">
	  	<img className="arrow-left" src="assets/arrow-left.svg" />
	  	<div className="compare-top">
	   		<h5>{loc1}</h5>
	   		<h4>average: {avg1}</h4>
	    </div>
	    <div>
			  <h3>{compare}</h3>
			  <h6>DIFFERENCE OF AVERAGES</h6>
		  </div>
	    <div className="compare-bottom">
		    <h4>average: {avg2}</h4>
		    <h5>{loc2}</h5>
	    </div>
	    <img className="arrow-right" src="assets/arrow-right.svg" />
	  </div>
	)
}

export default ComparisonCard;
