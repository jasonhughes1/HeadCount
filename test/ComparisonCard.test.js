import React from 'react';
import { shallow } from 'enzyme';
import ComparisonCard from './../src/Components/ComparisonCard';

describe('ComparisonCard Tests', () => {
  it('should match the snapshot', () => {
    const renderedApp = shallow(<ComparisonCard />)

    expect(renderedApp).toMatchSnapshot();
  })
})
