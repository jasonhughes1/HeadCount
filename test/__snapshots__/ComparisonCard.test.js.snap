exports[`ComparisonCard Tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ComparisonCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          className="arrow-left"
          src="assets/arrow-left.svg" />,
        <div
          className="compare-top">
          <h5 />
          <h4>
                    average: 
          </h4>
</div>,
        <div
          className="avg-diff">
          <h3 />
          <h6>
                    DIFFERENCE OF AVERAGES
          </h6>
</div>,
        <div
          className="compare-bottom">
          <h4>
                    average: 
          </h4>
          <h5 />
</div>,
        <img
          className="arrow-right"
          src="assets/arrow-right.svg" />,
      ],
      "className": "comparison-center",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "arrow-left",
          "src": "assets/arrow-left.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h5 />,
            <h4>
              average: 
</h4>,
          ],
          "className": "compare-top",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "h5",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "average: ",
                undefined,
              ],
            },
            "ref": null,
            "rendered": Array [
              "average: ",
              undefined,
            ],
            "type": "h4",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3 />,
            <h6>
              DIFFERENCE OF AVERAGES
</h6>,
          ],
          "className": "avg-diff",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "DIFFERENCE OF AVERAGES",
            },
            "ref": null,
            "rendered": "DIFFERENCE OF AVERAGES",
            "type": "h6",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h4>
              average: 
</h4>,
            <h5 />,
          ],
          "className": "compare-bottom",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "average: ",
                undefined,
              ],
            },
            "ref": null,
            "rendered": Array [
              "average: ",
              undefined,
            ],
            "type": "h4",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "h5",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "arrow-right",
          "src": "assets/arrow-right.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            className="arrow-left"
            src="assets/arrow-left.svg" />,
          <div
            className="compare-top">
            <h5 />
            <h4>
                        average: 
            </h4>
</div>,
          <div
            className="avg-diff">
            <h3 />
            <h6>
                        DIFFERENCE OF AVERAGES
            </h6>
</div>,
          <div
            className="compare-bottom">
            <h4>
                        average: 
            </h4>
            <h5 />
</div>,
          <img
            className="arrow-right"
            src="assets/arrow-right.svg" />,
        ],
        "className": "comparison-center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "arrow-left",
            "src": "assets/arrow-left.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h5 />,
              <h4>
                average: 
</h4>,
            ],
            "className": "compare-top",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h5",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "average: ",
                  undefined,
                ],
              },
              "ref": null,
              "rendered": Array [
                "average: ",
                undefined,
              ],
              "type": "h4",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3 />,
              <h6>
                DIFFERENCE OF AVERAGES
</h6>,
            ],
            "className": "avg-diff",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "DIFFERENCE OF AVERAGES",
              },
              "ref": null,
              "rendered": "DIFFERENCE OF AVERAGES",
              "type": "h6",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h4>
                average: 
</h4>,
              <h5 />,
            ],
            "className": "compare-bottom",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "average: ",
                  undefined,
                ],
              },
              "ref": null,
              "rendered": Array [
                "average: ",
                undefined,
              ],
              "type": "h4",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h5",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "arrow-right",
            "src": "assets/arrow-right.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
